py/nlrx64.c

```diff
@@ -35,6 +35,16 @@
 
 __attribute__((used)) unsigned int nlr_push_tail(nlr_buf_t *nlr);
 
+#if !MICROPY_NLR_OS_WINDOWS
+#if defined(__clang__) || (defined(__GNUC__) && __GNUC__ >= 8)
```

Comment:
nlrx86.c only uses "naked" for GCC >= 8 and never for Clang. However, Compiler Explorer suggests that the naked attribute works the same in Clang since 3.6 (2014).